clients/admin-ui/src/pages/privacy-requests/configure.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Box,
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  Heading,
+} from "@fidesui/react";
+import type { NextPage } from "next";
+import NextLink from "next/link";
+
+import Layout from "~/features/common/Layout";
+
+const ConfigurePrivacyRequests: NextPage = () => (
+  <Layout title="Configure Privacy Requests">
+    <Box mb={8}>
+      <Breadcrumb fontWeight="medium" fontSize="sm" color="gray.600">
+        <BreadcrumbItem>
+          <BreadcrumbLink as={NextLink} href="/privacy-requests">
+            Privacy requests
+          </BreadcrumbLink>
+        </BreadcrumbItem>
+        <BreadcrumbItem color="complimentary.500">
+          <BreadcrumbLink
+            as={NextLink}
+            href="/privacy-requests/configure"
+            isCurrentPage
+          >
+            Configuration
+          </BreadcrumbLink>
+        </BreadcrumbItem>
+      </Breadcrumb>
+    </Box>
+    <Heading mb={5} fontSize="2xl" fontWeight="semibold">
+      Configure your privacy requests
+    </Heading>
+    <Box display="flex" alignItems="center" mb={5}>
+      <NextLink href="/privacy-requests/configure/messaging" passHref>
+        <Box
```

Comment:
These should be [LinkBox](https://v1.chakra-ui.com/docs/components/navigation/link-overlay#) so they are actual anchor tags. The `cursor: "pointer"` should be unnecessary once it's the right element. (I bet we get this wrong in other places, fwiw).